RetailSuiteAccounting/UserInterface/01a_Bookings.md

```diff
@@ -0,0 +1,424 @@
+  This folder contains the registered documents. As soon as a new document is linked to a posting, it is moved automatically into the *Registered documents* folder and then placed in the appropriate subfolder according to the following structure:
+
+    - 10000-69999 personal account numbers in case of a debtors'/customers' document
+    - 70000-79999 personal account numbers in case of a creditors'/suppliers' document
+
+  - *Printed documents*  
+  This folder contains the printed documents.
+
+  - *New supplier receipts*  
+  This folder contains the newly uploaded supplier receipts.
+
+
+The buttons on top allow to navigate through the folders and documents.
+
+- [START PAGE]  
+Click this button to go back to the initial view of the *Attach document* window, see [Attach document](#attach-document).
+
+- [REFRESH]  
+Click this button to refresh the window, for example after making some changes.
+
+- [PARENT DIRECTORY]  
+Click this button to go directly from the current folder to its parent directory.  
+
+- [NEW DIRECTORY]  
+Click this button to create a new folder.
+
+- [UPLOAD]  
+Click this button to upload a new document. The *Upload file to* window is displayed, see [Upload file](#upload-file).
+
+- [DELETE]  
+Click this button to delete a selected document or folder.
+
+
+All information provided in columns is read-only.
+
+- [Checkbox]  
+[comment]: <> (Checkbox does not seem to be working; when selecting the checkbox in the column header, only that one and the last one are selected, not all, as it is usually. Function? Bug?)
+
+- *Name*  
+Folder or document name.
+
+- *Size*  
+Document size.
+
+- *Type*  
+Item type (folder).
+
+- *Changed*  
+Date when the items has been changed.
+
+- *Owner*  
+Item's owner.
+
+
+- [SELECT DOCUMENT]  
+Click this button to assign this document to a posting. The *Attached document* window is briefly displayed, see [Attached document](#attached-document).
+
+- [CLOSE]  
+Click this button to close the window.
+
+
+### Upload file
+
+*Accounting > Post > Tab POSTINGS > Button ASSIGN RECEIPT > Button UPLOAD*
+
+![Attach document window](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/UploadFileTo.png "[Attach document window]")
+
+
+The information related to the uploaded documents is displayed in column view. All fields are read-only.
+
+- *Status*  
+This column displays the document status.
+
+- *Document*  
+This column displays the document name together with its path.
+
+- *Remark*  
+This column displays any remarks related to the uploaded document.
+
+
+- [ADD]  
+Click this button to add a file to be uploaded.
+
+- [REMOVE]  
+Click this button to remove a selected file from the upload list.
+
+- [RESET]  
+Click this button to reset the upload list.
+
+- [UPLOAD]  
+The upload starts automatically after adding a file. This button changes to [CANCEL] during the upload.  
+
+- [CLOSE]  
+Click this button to close the window when the upload is finished.
+
+
```

Comment:
```suggestion
- [CANCEL]   
  Click this button to cancel the current upload. This button is only displayed during the upload. Otherwise, the [UPLOAD] button is displayed instead. 
```